conf/application.conf

```diff
@@ -118,11 +118,6 @@ controllers {
 
 # Metrics plugin settings - graphite reporting is configured on a per env basis
 metrics {
-  name = ${appName}
-  rateUnit = SECONDS
-  durationUnit = SECONDS
-  showSamples = true
-  jvm = true
   enabled = true
```

Comment:
I think this can be deleted too (i.e. the whole `metrics` section) because it's set in app-config-common.  I don't think we need metrics enabled in local dev (it's for Grafana if I'm not mistaken).

See https://github.com/hmrc/app-config-common/blob/b12e8a4fa3c85da52ff84a94955c042b8e8ef48e/development-microservice-common.yaml#L99

And see https://catalogue.tax.service.gov.uk/service/third-party-application/config#metrics.enabled